### ISSUE TITLE
Dear Vether

### ISSUE THREAD

**Issue by codomposer** [vethers/vethers#1]
I saw your work and I am interested in this project.
But there is not enough with review data.

**Comment by murongg** [comment]
Yes ! This project only just start ! I'm working hard to develop !

**Comment by walmartwarlord** [comment]
Hi, nice project. What's the difference between this and https://vagmi.vercel.app? Thank you

**Comment by murongg** [comment]
Oh! God! This project is exactly the same as my idea.

**Comment by murongg** [comment]
> Hi, nice project. What's the difference between this and https://vagmi.vercel.app? Thank you


Oh! God! This project is exactly the same as my idea.
